etc/config/c++.amazon.properties

```diff
@@ -412,6 +422,11 @@ compiler.avrg454.supportsBinary=false
 compiler.avrg464.exe=/opt/compiler-explorer/avr/gcc-4.6.4/bin/avr-g++
 compiler.avrg464.semver=4.6.4
 compiler.avrg464.objdumper=/opt/compiler-explorer/avr/gcc-4.6.4/bin/avr-objdump
+compiler.avrg540.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-g++
+compiler.avrg540.alias=avrg540
```

Comment:
Alias is used only when we need to fix typos or erros on compilers which have already gone live. There is no need to use it here :)